packages/cli/package.json

```diff
@@ -16,11 +16,15 @@
   },
   "scripts": {
     "lint": "eslint --ignore-path ../../.gitignore .",
-    "postbuild": "oclif-dev manifest"
```

Comment:
So the manifest is only for linking with the plugins plugin?

Reply:
The manifest is for telling OCLIF which commands the CLI / plugin contain. This CLI package not only doesn't export any commands itself, but it has no build step so the postbuild step never ran to generate the empty file to begin with.